cms-plugin/src/test/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/TestContentCategoryBulkAction.java

```diff
@@ -45,87 +37,97 @@
 class TestContentCategoryBulkAction extends ApsAdminBaseTestCase {
 
 	@Test
-    void testUserNotAllowed() throws Throwable {
+	void testUserNotAllowed() throws Throwable {
 		String[] contentIds = {"ART1", "RAH101", "EVN103"};
 		String[] categoryCodes = {"cat1"};
 		int strutsAction = ApsAdminSystemConstants.ADD;
 		String currentUser = "pageManagerCoach";
-		
+
 		String result = this.executeEntry(currentUser, strutsAction, contentIds);
 		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
-		
+
 		result = this.executeJoinCategory(currentUser, strutsAction, contentIds, categoryCodes, "evento");
 		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
-		
+
 		result = this.executeDisjoinCategory(currentUser, strutsAction, contentIds, categoryCodes, "evento");
 		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
-		
+
 		result = this.executeCheckApply(currentUser, strutsAction, contentIds, categoryCodes);
 		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
-		
+
 		result = this.executeApply(currentUser, strutsAction, contentIds, categoryCodes);
 		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
-		
-		result = this.executeCheckResult(currentUser, null);
-		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
-		
-		result = this.executeViewResult(currentUser, null);
+
+		result = this.executeCheckResult(currentUser);
 		assertEquals(BaseAction.USER_NOT_ALLOWED, result);
+// FIXME
```

Comment:
?

Reply:
I put a fixme to talk with @eugeniosant, i will remove it